kubernetes/omsagent.yaml

```diff
@@ -746,14 +746,23 @@ spec:
       port: 25227
       targetPort: in-rs-tcp
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: healthstates.azmon.container.insights
   namespace: kube-system
 spec:
   group: azmon.container.insights
-  version: v1
+  versions:
```

Comment:
since the helm chart as part of this code base, can you please also make these changes in the helm yamls so that we dont foget to add it? And also we will be using helm charts in our repo for the azure arc preview release.

Reply:
Sure I can add it. Also, could you add some documentation around how these charts are being released for non AKS cluster types like ARC?

Reply:
And how do i test the chart changes using the pipeline?

Reply:
> And how do i test the chart changes using the pipeline?

> just doing `helm install` command would work against your AKS or AKS-engine cluster.  Here is the link to pipeline https://github-private.visualstudio.com/microsoft/_releaseDefinition?definitionId=9&_a=definition-tasks&environmentId=22

Reply:
i will try this out. Also, will you be adding documentation around how these charts are being released for non AKS cluster types like ARC?

Reply:
> i will try this out. Also, will you be adding documentation around how these charts are being released for non AKS cluster types like ARC?

> currently this is done by the  pipelines for every release.  I have put some instructions here https://github.com/microsoft/Docker-Provider/blob/ci_dev/ReleaseProcess.md and need some updates to reflect current state of things as things are changing in Arc side, will try to make those doc updates tomorrow.

Reply:
> > i will try this out. Also, will you be adding documentation around how these charts are being released for non AKS cluster types like ARC?
> 
> > currently this is done by the  pipelines for every release.  I have put some instructions here https://github.com/microsoft/Docker-Provider/blob/ci_dev/ReleaseProcess.md and need some updates to reflect current state of things as things are changing in Arc side, will try to make those doc updates tomorrow.

> @rashmichandrashekar, I have updated the release notes in this PR https://github.com/microsoft/Docker-Provider/pull/450. Please review and let me know if you have any feedback.